### PR TITLE
Micro-manager user doc

### PR DESCRIPTION
**Pull request by hflynn**
This replaces https://github.com/openmicroscopy/bioformats/pull/2517

staged at https://www.openmicroscopy.org/site/support/bio-formats5.2-staging/users/micromanager/index.html

@bramalingam I'm afraid I'm not clear what the menu options at the bottom are for, perhaps you can clarify please

### CODE REVIEW

File: docs/sphinx/users/micromanager/index.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Micro-Manager
+=============
+
+`Micro-Manager <https://www.micro-manager.org/wiki/Micro-Manager>`_ is a
+software framework for implementing advanced and novel imaging procedures,
+extending functionality, customization and rapid development of specialized
+imaging applications.
+
+Micro-Manager offers the functionality for saving the acquired images in
+Tiff/OME-Tiff format. Based on the mode of saving and the configuration
```
Comment (sbesson) [inline docs/sphinx/users/micromanager/index.txt]: Could be unified to usig `TIFF` and `OME-TIFF` across the page

File: docs/sphinx/users/micromanager/index.txt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+|               |          |              | plus minimal |                    |
+|               | ome-tiff | No           | metadata*    | OMETiffReader      |
+| Save as image +----------+--------------+--------------+--------------------+
+| stack file    |          |              |              |                    |
+|               | ome-tiff | Yes**        | Full Support | MicromanagerReader |
++---------------+----------+--------------+--------------+--------------------+ 
+
+
+``*`` Minimal metadata: minimum amount of metadata required to validate
+OME-XML
```
Comment (sbesson) [inline docs/sphinx/users/micromanager/index.txt]: Discussed with @hflynn, we use `minimal` in a very specific semantics in the OME-TIFF specification documentation (minimal specification) and we might not want to confuse terms.

File: docs/sphinx/users/micromanager/index.txt [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+.. figure:: /images/micromanager_saving.png
+    :width: 50%
+    :align: center
+    :alt: Saving options in Micro-Manager
+
+
++---------------+----------+--------------+--------------+--------------------+
+|Saving         |          |Companion     |              |                    | 
+|Options within |          |File          |Bio-Formats   |                    |
+|Micro-Manager  |Format    |(metadata.txt)|Reading       | Reader Used        |
```
Comment (sbesson) [inline docs/sphinx/users/micromanager/index.txt]: `metadata.txt` -> `*metadata.txt`

**Comment by sbesson** [comment]
Reads well now. One question is whether we should link to this page in the Micro-manager format page with the additional information. This does not need to block this PR and can be carried out in a patch release.

**Comment by hflynn** [comment]
Micro-manager is at the opposite end of the format-pages.txt file to Zeiss CZI - this shouldn't conflict with the version history if I add to this PR, should it?

**Comment by sbesson** [comment]
No adding it to this PR should be fine.

**Comment by hflynn** [comment]
Right, will push another commit before the meeting then

**Comment by hflynn** [comment]
Done

**Comment by sbesson** [comment]
Looks good. Thanks.

**Pull request merged**